.github/workflows/unittests.yml

```diff
@@ -10,36 +10,37 @@
 
  jobs:
    unittests:
-     name: conda (${{ matrix.os }}, ${{ matrix.environment-file }})
+     name: CI (${{ matrix.os }}, ${{ matrix.environment-file }})
      runs-on: ${{ matrix.os }}
      timeout-minutes: 25
      strategy:
        matrix:
          os: ['macos-latest', 'ubuntu-latest', 'windows-latest']
-         environment-file: [ci/36.yaml, ci/37.yaml, ci/37-numba.yaml, ci/38.yaml, ci/38-numba.yaml]
-         exclude:
-           - environment-file: ci/36.yaml
-             os: windows-latest
+         environment-file: [ci/37.yaml, ci/37-numba.yaml, ci/38.yaml, ci/38-numba.yaml, ci/39.yaml, ci/39-numba.yaml]
      steps:
-       - uses: actions/checkout@v2
-       - uses: conda-incubator/setup-miniconda@v2
+       - name: checkout repo
+         uses: actions/checkout@v2
+       - name: setup micromamba
+         uses: mamba-org/provision-with-micromamba@main 
          with:
-            miniconda-version: 'latest'
-            auto-update-conda: true
-            auto-activate-base: false
+            micromamba-version: 'latest'
             environment-file: ${{ matrix.environment-file }}
-            activate-environment: test
-       - shell: bash -l {0}
-         run: conda info --all
-       - shell: bash -l {0}
-         run: conda list
-       - shell: bash -l {0}
-         run: conda config --show-sources
-       - shell: bash -l {0}
-         run: conda config --show
-       - shell: bash -l {0}
-         run: py.test esda --cov esda -v --cov-report=xml
-       - name: codecov (${{ matrix.os }}, ${{ matrix.environment-file }})
+       
+       - name: run tests - bash
+         shell: bash -l {0}
+         run: |
+           python -c 'import libpysal; libpysal.examples.fetch_all()'
```

Comment:
Are all the `libpysal` example datasets needed for the `esda` tests? If not, this can be removed (and the one in the chunk below.

Reply:
I do think some that are not built in are required. I think this needs to stay.

Reply:
OK!